resources/assets/js/editor.js

```diff
@@ -1,3 +1,5 @@
+const clientIsAuthor = document.querySelector(`meta[name='client-is-author']`).content === 'true';
```

Comment:
I don't think we need this as we already have a variable readOnly that tells us this information. Better not to double up.